### PR TITLE
Anchorectl changes

### PR DESCRIPTION
**Pull request by hev**
This PR makes some small changes to the AnchoreCTL page to improve clarity. 

* Changes name from anchore-cli to anchorectl in the title and nav
* Adds a sentence about overlap of anchore-cli
* Updates version number

Also includes changes in #308

### CODE REVIEW

File: content/docs/installation/anchorectl/_index.md [hunk 1]
```diff
@@ -16,7 +16,7 @@ Installation of AnchoreCTL (from a release) happens a few different ways, depend
```
Comment (Toure) [inline content/docs/installation/anchorectl/_index.md]: Just curious as to why the versions have been downgraded?

**Review by Toure — CHANGES_REQUESTED** [review]

File: content/docs/overview/concepts/images/inventory/_index.md [hunk 2]
```diff
@@ -1,153 +1,62 @@
...
 * For injecting the environment variable, see: `inject_secrets_via_env`
-* For providing your own secret for the Anchore API Password, see: `kai.existing_secret`. `kai` creates it's own secret based on your values.yaml file for key `kai.anchore.password`, but the `kai.existingSecret` key allows you to create your own secret and provide it in the values file.
\ No newline at end of file
+* For providing your own secret for the Anchore API Password, see: `kai.existing_secret`. `kai` creates it's own secret based on your values.yaml file for key `kai.anchore.password`, but the `kai.existingSecret` key allows you to create your own secret and provide it in the values file.
+
+## Agentless mode
+
+In addition to running as agent kai can also be run by Anchore Enterpise in an embedded mode. In this mode `kai` runs within the Anchore Enterprise, and needs a cluster configuration to access the Kubernetes API. This still requires cluster but allows for monitoring without the installation of additional components. See more details about setting up [embedded mode](./embedded/).
```
Comment (zhill) [inline content/docs/overview/concepts/images/inventory/_index.md:49]: This link should use the template format to ensure proper resolution and checks in CI: `({{< ref "./embedded" >}})`

File: content/docs/using/ui_usage/kubernetes/_index.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
+---
+title: "Kubernetes Inventory"
+linkTitle: "Kubernetes Inventory"
+weight: 1
+---
+Anchore Enterprise allows you to navigate through your Kubernetes clusters to quickly and easy asses your vulnerabilites, apply policies, and take action on them. You'll need to configure your clusters for collection before being able to take advanatage of these features. See our [installation instructions to get setup](/docs/overview/concepts/images/inventory/).
```
Comment (zhill) [inline content/docs/using/ui_usage/kubernetes/_index.md:6]: Similar ref update here.

**Review by zhill — CHANGES_REQUESTED** [review]
Overall looks good, but needs updates to the refs to use the template format so that, for example if they don't exist the hugo build will fail.

Also, since this updates some anchorectl download URLs, they need to all be consistent and reference 0.1.2. They are currently mixed in version.

**Comment by zhill** [comment]
> This PR makes some small changes to the AnchoreCTL page to improve clarity.
> 
> * Changes name from anchore-cli to anchorectl in the title and nav
> * Adds a sentence about overlap of anchore-cli
> * Updates version number
> 
> Also includes changes in #308

Should we close #308 then?

**Comment by zhill** [comment]
I'm going to merge this and make updates myself after merge as part of other updates.

**Pull request merged**